frontend/src/components/Worksheet/WorksheetCalendarList.tsx

```diff
@@ -51,12 +68,19 @@ export const StyledBtn = styled.div`
     background-color ${({ theme }) => theme.trans_dur},
     color ${({ theme }) => theme.trans_dur};
 
+  &:active {
+    background-color: ${({ theme }) => theme.button_active} !important;
+  }
+
+  &:disabled {
+    background-color: transparent;
+    color: ${({ theme }) => theme.text[2]} !important;
+  }
+
   &:hover {
     border: 2px solid hsl(0, 0%, 70%);
-    ${StyledBsEyeSlash} {
-      transform: scale(1.15);
-    }
-    ${StyledBsEye} {
+    background-color: ${({ theme }) => theme.button_active};
+    ${StyledBsEyeSlash}, ${StyledBsEye}, ${StyledTbFileExport} {
       transform: scale(1.15);
     }
   }
```

Comment:
we should add 

```
const StyledButtonGroup = styled(ButtonGroup)`
...
`;
```

here to change the background of the dropdown if the user is in dark mode. then use a `Dropdown` component with `as={StyledButtonGroup}`. anyway not urgent so approving

Reply:
see https://react-bootstrap-v4.netlify.app/components/dropdowns/#customization

tbh it's not even very clear if this would work. the docs are very unclear for v4, we should upgrade to BS v5 asap